pymchelper/detector.py

```diff
@@ -129,7 +131,7 @@ def average_with_other(self, other_detector, error_estimate=ErrorEstimate.stderr
         self.data += delta / self.counter                      # mean += delta / n
         if error_estimate != ErrorEstimate.none:
             self._M2 += delta * (other_detector.data - self.data)  # M2 *= delta * (x - mean)
```

Comment:
There is a typo in a comment to the right. It should be:
`M2 += delta * (x - mean)`
not
`M2 *= delta * (x - mean)`